static/app/views/performance/newTraceDetails/index.tsx

```diff
@@ -251,13 +313,21 @@ type TraceViewContentProps = {
   organization: Organization;
   traceEventView: EventView;
   traceSlug: string;
-  traces: TraceSplitResults<TraceFullDetailed> | null;
+  traceSplitResult: TraceSplitResults<TraceFullDetailed> | null;
 };
 
 function TraceViewContent(props: TraceViewContentProps) {
-  const {organization, traces, traceSlug, location, traceEventView, metaResults} = props;
+  const {
+    organization,
+    traceSplitResult,
+    traceSlug,
+    location,
+    traceEventView,
+    metaResults,
+  } = props;
 
-  const root = traces?.transactions?.[0];
+  const [traceType, setTraceType] = useState<TraceType | null>(null);
```

Comment:
Why does this need to be a state value and why are we initializing it from the child component? This seems like an anti pattern